src/esp/assets/ResourceManager.cpp

```diff
@@ -53,6 +57,7 @@
 #endif
 
 namespace Cr = Corrade;
+namespace Mn = Magnum;
```

Comment:
Thanks for this! 👍

Reply:
Updating the code as I go :) But only as long as it doesn't add unrelated changes to the diff.